journal/modernized/01-journal.adoc

```diff
@@ -2814,7 +2814,7 @@ I desired him to have a care what he said, and not lay the fault there:
 for it was the enemy, the adversary, the devil that begot that covetous mind in him,
 against his peaceable neighbours.
 I desired the Lord to forgive him; and as for me and my friends, we would forgive him;
-and I desired him to go his way, and to do so no more.
+and I desired him to go his way, and to do so no more.`'
```

Comment:
I think this closing single quote should go at the end of line 2812 instead, that seems to be where the informant stops speaking, I'm pretty sure.